docs/reference-guides/data/data-core-customize-widgets.md

```diff
@@ -50,11 +50,11 @@ Returns an action object used to open/close the inserter.
 _Usage_
 
 ```js
+import { useState } from 'react';
```

Comment:
Minor import sorting nit that I noticed throughout the changeset: isn't `react` supposed to be below the `@wordpress` imports if we sort alphabetically? Otherwise, if we're sorting by type of imports (external-wordpress-internal), should we include the import type comments to make that clearer?

Reply:
To be honest, I like keeping external imports first but I find the comments useless

Reply:
Fair enough 👍 No objections there.